imap_processing/codice/codice_l1a.py

```diff
@@ -89,28 +87,6 @@ def __init__(self, table_id: int, plan_id: int, plan_step: int, view_id: int):
         self.plan_step = plan_step
         self.view_id = view_id
 
-    def calculate_epoch_values(self) -> NDArray[int]:
```

Comment:
I moved this method outside of the class and made it a function so I can re-use it in multiple places in the code.